config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
 
+  get 'favourite_games/create'
+  get 'favourite_games/destroy'
```

Comment:
Don't think we need these two? Auto generated?

Reply:
I swear to god I deleted these!